src/misc.jl

```diff
@@ -4,8 +4,8 @@
 #
 # See: https://en.wikipedia.org/wiki/Multivariate_gamma_function
 #
-function logmvgamma(p::Int, a::Float64)
-    res::Float64 = p * (p - 1.0) / 4.0 * log(pi)
+function logmvgamma(p::Int, a::Real)
+    res = p * (p - 1.0) / 4.0 * log(pi)
```

Comment:
I was thinking of `Float32`. It seems wrong to promote to `Float64` if you allow `Real` input. You are right that `log(pi)` is `Float64` so I'm not completely sure how this should be handled. Maybe `log(pi*one(a))`?